lib/index.js

```diff
@@ -110,21 +138,29 @@ function GitHubLabeller (labels, options, callback) {
  * @param {String} owner The owner username.
  * @param {String} repo The repository name.
  * @param {Object} label The label object.
+ * @param {Array} label The list of current labels.
  * @param {Function} callback Callback function
  * @return {Request} The request object.
  */
-GitHubLabeller.addToRepo = function (ev, gh, owner, repo, label, callback) {
+GitHubLabeller.addToRepo = function (ev, gh, owner, repo, label, labels, callback) {
+    var endpoint = "repos/" + owner + "/" + repo + "/labels";
```

Comment:
@wooorm No, but it requires Node >= 4. I will do that in another version when rewriting this code. :joy: